pkgs/development/compilers/jsonnet/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchFromGitHub, emscripten }:
+
+let version = "0.8.5"; in
+
+stdenv.mkDerivation rec {
+  name = "jsonnet-${version}";
+
+  srcs = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "google";
+    repo = "jsonnet";
+    sha256 = "14raml69zfr38r4qghdgy129vdq2vq1ivl3a2y02isfpijxcajxn";
+  };
+
+  buildInputs = [ emscripten ];
+
+  enableParallelBuilding = true;
+
+  buildPhase = ''
+    make all EM_CACHE=$PWD/.em_cache
```

Comment:
Did you intend to overwrite `buildPhase`? Because overwriting it will render `enableParallelBuilding = true` pointless. You could pass EM_CACHE using `makeFlags`.